recipes-debian/thin-provisioning-tools/thin-provisioning-tools_debian.bb

```diff
@@ -0,0 +1,26 @@
+# base recipe: https://github.com/openembedded/meta-openembedded/tree/warrior/meta-oe/recipes-support/thin-provisioning-tools/thin-provisioning-tools_0.7.6.bb
```

Comment:
Could you change this line to the following?
`# base recipe: meta-oe/recipes-support/thin-provisioning-tools/thin-provisioning-tools_0.7.6.bb`